crates/dojo-core/src/world.cairo

```diff
@@ -564,6 +565,45 @@ mod world {
         }
     }
 
+    #[abi(embed_v0)]
+    impl UpgradeableState of IUpgradeableState<ContractState> {
+        fn upgrade_state(ref self: ContractState, new_state: Span<felt252>, program_output: Span<felt252>) {
+            let program_hash = 18468145346606952491117791430885172818429658569709932865894865254677766952;
```

Comment:
This may be for a next PR, but as we mentioned, let's try to have this configurable without re-declaring.

Reply:
yes, let's do that as another PR